Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -512,7 +512,7 @@ extension GridTabViewController: TabPeekDelegate {
     }
 
     func tabPeekRequestsPresentationOf(_ viewController: UIViewController) {
-        delegate?.tabTrayRequestsPresentationOf(viewController)
```

Comment:
Nit: It looks we could entirely remove `tabTrayRequestsPresentationOf` from the delegate.